OOP_Sample.rb

```diff
@@ -0,0 +1,56 @@
+class SalesTax
+  item = ""
+  total = 0
+  salesTax = 0
+  def generateSlip(str,n)
+    for i in 1..n
+      arr = str[i].split(" ")
+      quant = arr[0].to_i
+      price = arr[-1].to_f
+      if arr.length==4
+        item = arr[1]
+      else
+        for j in 1..arr.length-3
+          item += arr[j] + " "
+        end
+      end
+      getValue(quant, price, item)
+    end
+    puts "  Sales Taxes: #{salesTax.round(2)}"
+    puts "  Total: #{total.round(2)}"
+  end
+  def getValue(quant, price, item)
+    if item.include?("imported")
+      if item.include?("book") || item.include?("chocolate") || item.include?("headache pills")
+        val = ( quant * price * 105.00 / 100).round(2)
+        total += val
+        salesTax += ( val - price )
+        puts (" #{quant} #{item} : #{val}")
+      else
+        val = ( quant * price * 105.0 * 110.0 / 10000 ).round(2)
+        total += val
+        salesTax += ( val - price )
+        puts(" #{quant} #{item} : #{val}")
```

Comment:
These lines of code are repeating in `if` and `else` block. Just some data is changed in calculations. Can we refactor this code?